config/files/server/usr/etc/ssh/sshd_config.d/00-hardening.conf

```diff
@@ -1,17 +1,16 @@
+# Encryption hardening
+KexAlgorithms sntrup761x25519-sha512@openssh.com
+Ciphers aes256-gcm@openssh.com
```

Comment:
does this have any risk of breaking existing setups?

Reply:
No, you can break it up however you want.

Reply:
@TommyTran732 what do you mean?

Reply:
Oh, I just hallucinated and misread what you said. I am still sleepy haha. Ignore that comment.

No, I don't think it will break existing setups, so long as the clients support those. I know for a fact macOS and Fedora's SSH clients work with it.

Reply:
@TommyTran732 hmm. Normally it would be okay to merge this with a reasonable degree of certainty since users can always roll back on desktop.

But on a server, loss of ssh access would prevent rolling back, so we need to be extra careful here